tests/integration/test_scmsteprun_view.py

```diff
@@ -158,3 +158,51 @@ def test_create(self, client, logged_in_user, scm_pipeline_run, scm_step_run):
         assert response.status_code == 201
         assert models.SCMStepRun.objects.filter(name='Release product').exists()
         assert models.SCMStepRun.objects.count() == initial_count + 1
+
+    def test_status_update_with_boolean(self, client, logged_in_user, scm_step_run):
+        assert scm_step_run.status == 'not started'
+
+        url = f'/scm-step-runs/{scm_step_run.public_identifier}/status_success/'
+        data = {'status_success': True}
+        response = client.patch(url, data, content_type='application/json')
+        assert response.status_code == 204
+        p = models.SCMStepRun.objects.get(pk=scm_step_run.public_identifier)
+        assert p.status == 'success'
+
+    def test_status_update_with_string(self, client, logged_in_user, scm_step_run):
+        assert scm_step_run.status == 'not started'
+
+        url = f'/scm-step-runs/{scm_step_run.public_identifier}/status_success/'
+        data = {'status_success': 'yes'}
```

Comment:
(Why) should this work? Should the request with JSON not only allow Boolean values?

Reply:
that's the default behaviour of DRF boolean field.

Reply:
Oh wow.. I checked the docs of both DRF and Django and did not see any mention of this but if it is default behaviour let's keep it that way

Reply:
yeah, i saw in the source code:
https://github.com/encode/django-rest-framework/blob/master/rest_framework/fields.py#L668
Just added the test to make visible that works like that